firebase-installations/src/main/java/com/google/firebase/installations/local/IidStore.java

```diff
@@ -38,17 +40,23 @@
   private static final String IID_SHARED_PREFS_NAME = "com.google.android.gms.appid";
   private static final String STORE_KEY_PUB = "|S||P|";
   private static final String STORE_KEY_ID = "|S|id";
+  private static final String STORE_KEY_TOKEN = "|T|";
+  private static final String DEFAULT_SCOPE = "|*";
+  private static final String JSON_TOKEN_KEY = "token";
```

Comment:
Can you please add some comments here:
What do all these values mean? Where do the values come from? What is a scope? Where do you use JSON_TOKEN_KEY and so on...

Example:
// The Instance-ID SDK creates a wildcard-scoped ("*") Registration-IID-Token by default.
// The vertical bar is used to separate information identifying Registrations and IID-Tokens.